_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -38,25 +38,25 @@ At a high level, Scala shares these *similarities* with Python:
 
 - Both are high-level programming languages, where you don’t have to concern yourself with low-level concepts like pointers and manual memory management
 - Both have a relatively simple, concise syntax  
-- Both are functional programming (FP) languages
+- Both are [functional programming][fp-intro] (FP) languages
```

Comment:
Maybe say: "support a functional style of programming" like in most other places. Otherwise this triggers controversial discussions.